### PR TITLE
Direction for Windows-developer mode

### PR DESCRIPTION
**Pull request by yias**
[sc-16695]

This PR adds install directions on Windows for the developer track

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -69,21 +69,51 @@ need to follow these steps:
...
+
+         .. code:: shell
+
+            python -m pip install tox
+
+      * and then run tox
+
+         .. code:: shell
+
+            tox
```
Comment (tmpbeing) [inline README.rst:115]:
```suggestion
   .. code:: bash

         python -m pip install tox
         tox
```

Trying to reduce noise by assuming the user has pip by that point and these commands work for both systems. If the user doesn't have pip I'm sure the error message and google will be enough.

File: README.rst [hunk 2]
```diff
@@ -69,21 +69,51 @@ need to follow these steps:
 
       git clone https://github.com/ansys/pysimai
 
-#. `Install PDM <https://pdm-project.org/latest/#recommended-installation-method>`_
+#. `Install PDM <https://pdm-project.org/latest/#recommended-installation-method>`_. NB: If you are a Windows user, make sure that Python is installed on your system and it is added to the Path.
```
Comment (tmpbeing) [inline README.rst:72]:
```suggestion
#. `Install PDM <https://pdm-project.org/latest/#recommended-installation-method>`_.
```

I think we should assume developers have python properly installed on their system.

**Review by tmpbeing — APPROVED** [review]
Thanks. Made some suggestion to try and keep it less verbose.
Eventually we'll have to move some things to a contribute section of the doc.

File: README.rst (hunk 2 above)
Comment (awoimbee) [reply README.rst:72]: Just saw this PR.
So if I'm not a windows user I don't need to have "Python is installed on your system and it is added to the Path" ?

**Pull request merged**